gooddata-dbt/gooddata_dbt/dbt_plugin.py

```diff
@@ -32,30 +33,36 @@ def layout_model_path(data_product: GoodDataConfigProduct) -> Path:
 
 def generate_and_put_ldm(
     logger: logging.Logger,
-    sdk: GoodDataSdk,
+    sdk_wrapper: GoodDataSdkWrapper,
     data_source_id: str,
     workspace_id: str,
     dbt_tables: DbtModelTables,
     model_ids: Optional[List[str]],
+    dry_run: bool = False,
```

Comment:
Is dry_run necessary? What happens if sdk_wrapper has dry_run=True and this default (False) is used? Does it mind that it can occur that these will not be in sync?

Reply:
fixed